src/lib.rs

```diff
@@ -1503,3 +1516,7 @@ impl<T: Counter, F: Counter> PartialEq<Histogram<F>> for Histogram<T>
 #[path = "tests/tests.rs"]
 #[cfg(test)]
 mod tests;
+
+/// Serialization/deserialization support.
+#[path = "serialization/serialization.rs"]
```

Comment:
Hopefully you'll come to enjoy it as I have (I was introduced to it on other projects I contribute to, and it felt weird to me at first as well). If it really bugs you I won't shed tears if you want it gone later. :)